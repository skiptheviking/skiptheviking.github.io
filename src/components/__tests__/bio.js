import React from "react"
import renderer from "react-test-renderer"

import Bio from "../bio"

// describe("Header", () => {
//     it("renders correctly", () => {
//         const tree = renderer
//             .create(<Header siteTitle="Default Starter" />)
//             .toJSON()
//         expect(tree).toMatchSnapshot()
//     })
// });

describe("Bio", ()=> {
    it("renders correctly", () => {
        const tree = renderer
            .create(<Bio/>)
            .toJSON()
        expect(tree).toMatchSnapshot()

    })
});