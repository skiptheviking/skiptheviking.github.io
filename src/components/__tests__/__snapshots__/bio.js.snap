// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bio renders correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "marginBottom": "4.375rem",
    }
  }
/>
`;
